src/pages/accessDenied.vue

```diff
@@ -4,13 +4,20 @@
           <h1 class="oc-login-logo" v-translate>
               ownCloud
           </h1>
-          <div class="oc-login-card-body">
+          <div class="oc-login-card-body uk-width-large">
               <h3 class="oc-login-card-title">
                   <span v-translate>Login Error</span>
               </h3>
               <h4 v-translate class="uk-margin-remove">
                 You are not allowed to use this application.
               </h4>
+              <br>
+              <div v-translate class="uk-margin-remove uk-text-break">
```

Comment:
Since there is the fixed width now I think it would be better to drop the `uk-text-break`. We can now simply break before the whole word. IMHO better solution.

Reply:
hmm .. I see no difference after removing uk-text-break ... well then we go in this direction :+1: